src/editor/components/modals/revision-diff.jsx

```diff
@@ -29,11 +48,40 @@ class RevisionDiffUnconnected extends React.Component {
     }
     return (
       <DiffContainer>
+        <RevertModal
+          visible={this.state.pendingRevert !== false}
+          onCloseOrCancel={() => {
+            this.setState({ pendingRevert: false });
+          }}
+          onRevert={() => {
+            this.props.revertToSelectedRevisionId();
+          }}
+          date={this.props.date}
```

Comment:
If you follow my suggestion, this would then be: `this.props.selectedRevision.created`

Reply:
Just needed a tertiary operation to prevent undefined access